### PR TITLE
Add 5 questions to QuestionBank

### PR DESCRIPTION
**Pull request by joelngyx**
Added 5 questions to the JSON file

Resolves part of #74

### CODE REVIEW

File: data/main/topics.json [hunk 1]
```diff
@@ -211,5 +211,120 @@
         ]
       }
     ]
+  },
+  {
+    "topic": "Java Basics",
```
Comment (kstonekuan) [inline data/main/topics.json]: Sorry need this to be one word something like Java_Basics for the parser

File: data/main/topics.json [hunk 2]
```diff
@@ -211,5 +211,120 @@
         ]
       }
     ]
+  },
+  {
+    "topic": "Java_Basics",
```
Comment (kstonekuan) [inline data/main/topics.json:397]: Thanks for changing this

**Review by kstonekuan — APPROVED** [review]
LGTM

**Review by Zhi-You — APPROVED** [review]
LGTM!

**Pull request merged**